### PR TITLE
pass options to GET parameters #10

### PR DESCRIPTION
**Pull request by limpbrains**
fixing issue #10

### CODE REVIEW

**Comment by ryandotsmith** [comment]
@limpbrains Thanks for the pull request! This is definitely a must-have.

File: http-utility.js [hunk 1]
```diff
@@ -43,13 +46,18 @@ HttpUtility.prototype.makeRequest = function(method, path, body,  cb) {
...
 };
 
 HttpUtility.prototype.delete = function(path, cb) {
-  this.makeRequest('DELETE', path, null, cb);
+  this.makeRequest('DELETE', path, null, null, cb);
 };
 
-HttpUtility.prototype.get = function(path, cb) {
-  this.makeRequest('GET', path, null, cb);
+HttpUtility.prototype.get = function(path, options, cb) {
```
Comment (ryandotsmith) [inline http-utility.js]: What do you think about renaming `options` to `query` or `params`? And whichever we land on, we should use the same name throughout the file.

**Comment by Levino** [comment]
As in the official documentation the object is called "options", I would stick with it.

bildschirmfoto vom 2015-01-05 11 44 38

**Comment by limpbrains** [comment]
I've thought the same, called it the same way you have in the documentaion.

**Comment by Levino** [comment]
@ryandotsmith Could you merge and release a new version? It is obviously an annoying bug :)

**Comment by ryandotsmith** [comment]
Can't argue with the docs...

This patch is now available in `3.0.1`.

**Pull request merged**